pages/unit/_unit.vue

```diff
@@ -10,32 +39,46 @@
 export default {
     name: "Unit",
     data() {
-        return {
-        }
+        return {}
     },
     created() {
-
     },
-    async mounted() {
-        await this.getUnit();
+    mounted() {
+        if (this.seasonInfo) {
+            this.getUnit()
+        }
     },
     beforeDestroy() {
     },
+    computed: {
+        seasonInfo() {
+            return this.$store.getters['seasonInfo'];
+        },
+        units() {
+            return this.$store.getters['units'];
```

Comment:
Codacy has a fix for the issue: [Replace `'units'` with `"units"`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9690065)
```suggestion
            return this.$store.getters["units"];
```